fastcov.py

```diff
@@ -60,7 +60,7 @@ def stopwatch():
 
 def parseVersionFromLine(version_str):
     """Given a string containing a dotted integer version, parse out integers and return as tuple"""
-    version = re.search(r'(\d+\.\d+\.\d+)[^\.]', version_str)
+    version = re.search(r'(\d+\.\d+\.\d+)[^\.]?', version_str)
```

Comment:
Making the last character optional kind of defeats the purpose of `[^\.]`. I suggest just getting rid of it altogether for now.
```suggestion
    version = re.search(r'(\d+\.\d+\.\d+)', version_str)
```